flux-fixpoint/src/constraint.rs

```diff
@@ -543,6 +543,66 @@ impl fmt::Display for Constant {
 impl Constant {
     pub const ZERO: Constant = Constant::Int(Sign::Positive, 0);
     pub const ONE: Constant = Constant::Int(Sign::Positive, 1);
+
+    fn to_bool(&self) -> Option<bool> {
```

Comment:
`Constant` is `Copy` so we can pass a `self` here. This also applies for all the other methods and places we take `&Constant`